### ISSUE TITLE
Changing the timezone in VShell

### ISSUE THREAD

**Issue by jahunter** [NagiosEnterprises/nagiosvshell#16]
I've tried adding `SetEnv TZ "Europe/London"` to the vshell.conf file but I'm still seeing times reflected as UTC.  Is there somewhere else that I need to make a change and if so where?

**Comment by mguthrie88** [comment]
Can you verify that your linux system time and your timezone in /etc/php.ini are consistent with that timezone?

**Comment by jahunter** [comment]
There is no timezone set in the php.ini file.  Also if I type 'date' at the command line I get this:
```
[root@server]# date
Thu Jun 30 13:31:38 BST 2016
```
so the server time looks OK.  So where else could vshell be getting the UTC time from?

**Comment by jahunter** [comment]
The good news is that this is no longer an issue.
The bad news is I don't really know why.

I restarted Apache after making the php.ini change, and restarted Nagios core.  Nothing I did seemed to have any effect.  So I finally resorted to the old IT adage of switching off and back on again.  A reboot of the server finally did the trick, but goodness only knows where the UTC time was being stored and referenced by Vshell.  I can only imagine it was cached in memory somewhere!!!!
